Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationItem.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Backpack/Button.h>
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * `BPKHorizontalNavigationItem` is a subclass of `UICollectionViewCell`.
+ */
+@class BPKHorizontalNavigationOption;
+NS_SWIFT_NAME(HorizontalNavigationItem) IB_DESIGNABLE @interface BPKHorizontalNavigationItem : BPKButton
```

Comment:
Not sure about inherting from `BPKButton` here, it feels like much more potential hassel than it's worth

Reply:
Do you reckon we should go for a custom `UIButton` then?

Reply:
Yeah that sounds reasonable

Reply:
tbh I kinda feel like using our button might make sense given that we want to support icons too...

Reply:
I'll give it a go and see where I get to!